.github/actions/awx_devel_image/action.yml

```
@@ -11,18 +11,24 @@ runs:
       shell: bash
       run: echo py_version=`make PYTHON_VERSION` >> $GITHUB_ENV
 
+    - name: Set lower case owner name
+      shell: bash
+      run: echo "OWNER_LC=${OWNER,,}" >>${GITHUB_ENV}
```

Just for style consistency

```suggestion
      run: echo "OWNER_LC=${OWNER,,}" >> $GITHUB_ENV
```